pkg/graph/schema.resolvers.go

```diff
@@ -440,11 +444,24 @@ func (r *mutationResolver) CreateAccessibilityRequest(ctx context.Context, input
 		return nil, err
 	}
 
-	request, err := r.store.CreateAccessibilityRequestAndInitialStatusRecord(ctx, &models.AccessibilityRequest{
+	newRequest := &models.AccessibilityRequest{
 		EUAUserID: requesterEUAID,
 		Name:      input.Name,
 		IntakeID:  input.IntakeID,
-	})
+	}
+
+	cedarSystemID := null.StringFromPtr(input.CedarSystemID)
+	cedarSystemIDStr := cedarSystemID.ValueOrZero()
+	if input.CedarSystemID != nil && len(*input.CedarSystemID) > 0 {
```

Comment:
Can you clarify this `if` statement & block?

It looks like if we pass a defined string that isnt "", that we'll query CEDAR and return an error if we get one, but it doesn't look like the success case actually sets `newRequest.CedarSystemID` properly.

I tested this with the following:
- Run `scripts/dev db:clean && scripts/dev db:seed`
- Grab the `id` from the `system_intakes` table for the request with name `With LCID Issued`
- Run the following mutation, using the ID above:
```graphql
mutation {
  createAccessibilityRequest(input: {
    intakeID: "YOUR_ID_HERE"
    name: "MY TEST 508 LOCAL"
    cedarSystemId: "326-689-0"
  }) {
    accessibilityRequest {
      id
      name
      cedarSystemId
    }
    userErrors {
      message
      path
    }
  }
}
```

What I get back has `cedarSystemId` set to `null` still.